### DISCUSSION TITLE
Is it possible to have multiple tunnel with same cloudflare account?

### DISCUSSION THREAD

**Discussion by Klagio** [homeassistant-apps/app-cloudflared#597] [Ideas]
When I try I get this. Maybe possible to give a name? 

[18:55:22] FATAL: Failed to create tunnel.
    Please check the Cloudflare Zero Trust Dashboard for an existing tunnel with the name homeassistant and delete it:
    Visit https://one.dash.cloudflare.com, then click on Access / Tunnels

**Comment by elcajon** [comment]
What are you trying to achieve? Creating two different tunnel for two HA installations? Or running two tunnels within one installation?

**Reply by HuckleberryLovesYou** [reply]
Not the author, but I'm trying to use the Cloudflared Homeassistant addon with my HA-Supervised-Installation to connect multiple services to cloudflare. Those Services are running on their own Docker Container. I'm talking about Services like Uptimekuma, Speedtesttracker and dockge. In my Expectations I would like to have multiple subdomains, which I already created, to connect to diffrent Services via cloudflared opened tunnels. Is there a way to use the cloudflared HA-addon instead of using the non-HA-application?

**Reply by elcajon** [reply]
Take a look at the [`additional_hosts`](https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#option-additional_hosts) add-on option.
